mappings/oclcCatalog.py

```diff
@@ -160,17 +160,10 @@ def applyFormatting(self):
         self.record.source = 'oclcCatalog'
         self.record.source_id = self.record.identifiers[0]
         self.record.frbr_status = 'complete'
-
-        logger.info(f'Formatting OCLC Catalog record with id: {self.record.source_id}')
```

Comment:
I definitely agree conceptually with removing most of the debug logging in this file, but many of the errors currently being thrown by the clustering process are difficult to identify because it's harder to get a handle on the original record (and which source is throwing the error) when the error passes through the mapping. I definitely think we should focus some energy on making it easier to pinpoint these issues before we re-introduce the ingest processes.

Reply:
makes sense!